### PR TITLE
Portfolio

### PR DESCRIPTION
**Pull request by Samuelcrudge**
finished portfolio.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -32,7 +33,59 @@ <h3>About me</h3>
...
+					</li>
+					<li  class="skills6">
+						<i class="fab fa-react"></i>
+					</li>
+				</ul>
+			</div>
+			<div class="projectlist">
+				<div class="container3">
+					<img src="assets/Project1.png">
+					<a href="_blank" target="_blank">
```
Comment (bradleypurdue) [inline index.html]: Is `href` meant to be `_blank`, is this a placeholder for now? Same applies for the rest below.

File: portfolio.css [hunk 2]
```diff
@@ -65,26 +65,92 @@ main {
...
+.skills5 :hover {
+	color: #95cb48;
+}
+.skills6 :hover {
+	color: #80d9f7;
+}
+.projectlist {
+	float: left;
+	width: 100%;
+	margin-top:7 0px;
```
Comment (bradleypurdue) [inline portfolio.css]: Missing units on `7`.

File: index.html [hunk 3]
```diff
@@ -32,7 +33,59 @@ <h3>About me</h3>
...
+						<i class="fab fa-node"></i>
+					</li>
+					<li  class="skills6">
+						<i class="fab fa-react"></i>
+					</li>
+				</ul>
+			</div>
+			<div class="projectlist">
+				<div class="container3">
+					<img src="assets/Project1.png">
```
Comment (bradleypurdue) [inline index.html]: Missing `alt` attribute on the ``.

File: index.html [hunk 4]
```diff
@@ -32,7 +33,59 @@ <h3>About me</h3>
...
+					<a href="_blank" target="_blank">
+						<button class="button1">Code</button>
+					</a>
+					<a href="_blank" target="_blank">
+						<button class="button2">View</button>
+					</a>
+				</div>
+				<div class="container3">
+					<img src="assets/Project2.png">
```
Comment (bradleypurdue) [inline index.html]: Missing `alt` attribute on the ``.

File: index.html [hunk 5]
```diff
@@ -32,7 +33,59 @@ <h3>About me</h3>
...
+					<a href="_blank" target="_blank">
+						<button class="button1"> Code</button>
+					</a>
+					<a href="_blank" target="_blank">
+						<button class="button2">View</button>
+					</a>
+				</div>
+				<div class="container3">
+					<img src="assets/Project3.png">
```
Comment (bradleypurdue) [inline index.html]: Missing `alt` attribute on the ``.

File: portfolio.css [hunk 6]
```diff
@@ -65,26 +65,92 @@ main {
...
+	background-color:#acacac;
+}
+.button1:hover {
+	background-color:#ffd944;
+}
+.button2:hover {
+	background-color:#dd4719cf;
+}
+.projectlist img {
+	display: block;
```
Comment (bradleypurdue) [inline portfolio.css]: Indentation is off here.

File: portfolio.css [hunk 7]
```diff
@@ -65,26 +65,92 @@ main {
...
+	text-align: center;
+	font-family: 'Oxanium','Times New Roman';
+}
+.container3 button {
+    margin-left: 128px;
+    width: 83px;
+    height: 30px;
+    margin-top: 20px;
+	font-size:2rem;
+	background-color:#acacac;
```
Comment (bradleypurdue) [inline portfolio.css]: Indentation is a bit off.

File: index.html (hunk 1 above)
Comment (Samuelcrudge) [reply index.html]: placeholder

**Pull request merged**